kedro/framework/cli/pipeline.py

```diff
@@ -549,10 +572,10 @@ def _install_files(  # pylint: disable=too-many-locals
     )
 
     if conf_source.is_dir() and alias:
-        alias_name = alias.split(".")[-1]
-        _rename_files(conf_source, package_name, alias_name)
+        _rename_files(conf_source, package_name, alias)
 
-    module_path = alias or package_name
+    destination = destination or ""
+    module_path = f"{destination}.{alias or package_name}"
```

Comment:
ok, I see this is used later on as
```
    artifacts = (
        Path(package_dir, *module_path.split(".")),
        Path(package_dir.parent, "tests", *module_path.split(".")),
        project_conf_path / env,
    )
```
and so if `destination = ""` then I'll get `module_path = ".alias"` which after splitting becomes `["", "alias"]`, which works ok with `Path`. And if `destination = "something/nested"` it will also work ok because I'll get `module_path = "something/nested.alias"`, which after splitting becomes `["something/nested", "alias"]`, which also works ok with `Path`.

Is that right? Seems kind of confusing either way. Can we simplify this somehow? 😅

Reply:
Yes, exactly how it worked. I will have a look to see if we can simplify this.